torch/_dynamo/debug_utils.py

```diff
@@ -782,3 +784,35 @@ def gen_tensor(shape, dtype) -> Tensor:
             setattr(container, attr_name, gen_tensor(shape, dtype))
 
     return kwargs
+
+
+def profile_to_file(filename: str) -> Callable[[T], T]:
```

Comment:
maybe leave a doc block explaining what it does and how to use it

although the function is short enough that maybe not necessary